sdk/client.go

```diff
@@ -31,16 +34,34 @@ func NewWithHealthClient(hcCli *health.Client) *Client {
 }
 
 // URL returns the URL used by this client
-func (cli *Client) URL() string {
-	return cli.hcCli.URL
+func (c *Client) URL() string {
+	return c.hcCli.URL
 }
 
 // Health returns the underlying Healthcheck Client for this API client
-func (cli *Client) Health() *health.Client {
-	return cli.hcCli
+func (c *Client) Health() *health.Client {
+	return c.hcCli
 }
 
 // Checker calls topic api health endpoint and returns a check object to the caller
-func (cli *Client) Checker(ctx context.Context, check *healthcheck.CheckState) error {
-	return cli.hcCli.Checker(ctx, check)
+func (c *Client) Checker(ctx context.Context, check *healthcheck.CheckState) error {
+	return c.hcCli.Checker(ctx, check)
+}
+
+// Adds collectID to request header if not empty
+// TODO: Add to dp-net?
```

Comment:
I would suggest removing this TODO for now as I think if it's something we're going to do it will be later, and this TODO suggests the code isn't finished yet.